### PR TITLE
RDRP-767: tidy schemas

### PR DESCRIPTION
**Pull request by JenCheshire**
Tidied schemas to match DAP

## Pull Request submission

*Insert detailed bullet points about your changes here!*

*Insert any instructions to help the reviewer, e.g. "install new requirements from `environment.yml`"*

*Let the reviewer know what data files are needed (and if applicable, where they are to be found)

#### Closes or fixes

* Detail the ticket(s) you are closing with this PR
Closes #

#### Code

- [ ] **Code runs** The code runs on my machine and/or CDSW
- [ ] **Conflicts resolved** There are no conflicts (I have performed a rebase if necessary)
- [ ] **Requirements** My/our code functions according to the requirements of the ticket
- [ ] **Dependencies** I have updated the environment yaml so it includes any new libraries I have used
- [ ] **Configuration file updated** any high level parameters that the user may interact with have been put into the config file (and imported to the script)
- [ ] **Clean Code**
    - [ ] Code is as [PEP 8]([url](https://peps.python.org/pep-0008/)) compliant as I can humanly make it
    - [ ] Code passess flake8 linting check
    - [ ] Code adheres to [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)
- [ ] **Type hints** All new functions have [type hints ](https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html)

#### Documentation

Any new code includes all the following forms of documentation:

- [ ] **Function Documentation** Docstrings within the function(s')/methods have been created
    - [ ] Includes `Args` and `returns` for all major functions
    - [ ] The docstring details data types
- [ ] **Updated Documentation**: User and/or developer working doc has been updated

#### Data
- [ ] All data needed to run this script is available in Dev/Test
- [ ] All data is excluded from this pull request
- [ ] Secrets checker pre-commit passes

#### Testing
- [ ] **Unit tests** Unit tests have been created and are passing _or a new ticket to create tests has been created_

---

# Peer Review Section

- [ ] All requirements install from (updated) `environment.yaml`
- [ ] Documentation has been created and is clear - **check the working document**
- [ ] Doctrings ([Google format](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)) have been created and accurately describe the function's functionality
- [ ] Unit tests pass, or if not present _a new ticket to create tests has been created_
- [ ] **Code runs** The code runs on reviewer's machine and/or CDSW

#### Final approval (post-review)

The author has responded to my review and made changes to my satisfaction.
- [ ] **I recommend merging this request.**

---

### Review comments

*Insert detailed comments here!*

These might include, but not exclusively:

- bugs that need fixing (does it work as expected? and does it work with other code
  that it is likely to interact with?)
- alternative methods (could it be written more efficiently or with more clarity?)
- documentation improvements (does the documentation reflect how the code actually works?)
- additional tests that should be implemented (do the tests effectively assure that it
  works correctly?)
- code style improvements (could the code be written more clearly?)
- Do the changes represent a change in functionality so the version number should increase? Start a discussion if so.
- As a review you can generates the same outputs from running the code

Your suggestions should be tailored to the code that you are reviewing.
Be critical and clear, but not mean. Ask questions and set actions.

### CODE REVIEW

**Review by AnneONS — COMMENTED** [review]
Hi Jen,

This looks good, it's all a bit confusing isn't it? I was able to run the code ok on the network with several outputs turned on.

I'm comparing pulled code in network and DAP, and in DAP you have 
intram_by_pg_schema,

whereas in github there's a _gb and _uk version. I think we do need both updated versions instead of the old one, cos the export code needs the name of the schema to match the name of the file.

Also, was there a decision that they'll never want to export intram by C or D?

In DAP, I can see we have an output schema for estimation_qa in the develop branch, but not in your branch.

File: config/output_schemas/short_form_schema.toml
Comment (AnneONS) [inline config/output_schemas/short_form_schema.toml:92]: I'm interested to know where these changes came from, were they on DAP?

File: config/wide_responses.toml
Comment (AnneONS) [inline config/wide_responses.toml]: can you check? 200 and 201 can be null, can't they?

**Review by AnneONS — COMMENTED** [review]
Just a couple of questions, looks good but will also check DAP a bit later.

File: config/output_schemas/short_form_schema.toml
Comment (JenCheshire) [reply config/output_schemas/short_form_schema.toml:92]: Yes, this was updated to reflect DAP

File: config/wide_responses.toml
Comment (JenCheshire) [reply config/wide_responses.toml]: Sure - @zorge69 are you able to confirm, or shall I go to BAU?  200 and 201 are Nullable = "No" in DAP

Comment (AnneONS) [reply config/wide_responses.toml]: I think we know that 200 and 201 are null for instance  0, for example. Also where there is no response. So I'd say it's "Yes", and we can update in DAP.

Comment (JenCheshire) [reply config/wide_responses.toml]: Great, that's been updated in both

**Review by AnneONS — APPROVED** [review]
This will be ready to merge now, or after 777 is merged.

**Pull request merged**
